modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -108,26 +123,18 @@ def _read(cls, filepath_or_buffer, **kwargs):
             parse_dates=parse_dates,
             usecols=usecols,
         )
-        encoding = kwargs.get("encoding", None)
         quotechar = kwargs.get("quotechar", '"').encode(
             encoding if encoding is not None else "UTF-8"
         )
         is_quoting = kwargs.get("quoting", "") != csv.QUOTE_NONE
         with cls.file_open(filepath_or_buffer, "rb", compression_type) as f:
-            # Skip the header since we already have the header information and skip the
-            # rows we are told to skip.
-            if isinstance(skiprows, int) or skiprows is None:
-                if skiprows is None:
-                    skiprows = 0
-                header = kwargs.get("header", "infer")
-                if header == "infer" and kwargs.get("names", None) is None:
-                    skiprows += 1
-                elif isinstance(header, int):
-                    skiprows += header + 1
-                elif hasattr(header, "__iter__") and not isinstance(header, str):
-                    skiprows += max(header) + 1
-            if kwargs.get("encoding", None) is not None:
-                partition_kwargs["skiprows"] = 1
```

Comment:
It is used, `skiprows` parameter defined inside `partitioned_file` function [here](https://github.com/modin-project/modin/pull/2607/files#diff-92c57d660c9f3f4ad28fdd6ebd1b3aafe688e79eea03f86d78b972930e840832R220-R224) and then passed to the `deploy` function [here](https://github.com/modin-project/modin/pull/2607/files#diff-172c7ff4cc90a4dd253fa690e0165e9ae1d8a79214bdde6a1def95546be7cb95R211). `skiprows` definition replaced because this parameter should be recalculated for each split in accordance to already `considered` rows.